Formula/c/cdo.rb

```diff
@@ -27,17 +27,26 @@ class Cdo < Formula
   depends_on "proj"
   uses_from_macos "python" => :build
 
+  on_macos do
+    depends_on "llvm" if DevelopmentTools.clang_build_version <= 1403
+  end
+
+  fails_with :clang do
+    build 1403
+    cause "Requires C++20 support"
+  end
+
   def install
+    ENV.llvm_clang if OS.mac? && DevelopmentTools.clang_build_version <= 1403
+
     args = %W[
```

Comment:
It automatically linked to OpenMP due to LLVM. We can disable that with:

```suggestion
    args = %W[
      --disable-openmp
```